### PR TITLE
adds two new Coalition voidsuits

### PR DESCRIPTION
**Pull request by crunchypretzels**
big PR, big problems. little pr? little problems

This PR adds two new voidsuits, the Galatean Jackdaw and the Himean Buzzard, modelled after the Vulture.

The Jackdaw is weaker to ballistics and melee, but easily resists radiation and biological damage, as well as energy damage (tesla cannons, blasters, and wizards).

The Buzzard is stronger against explosions and melee, but weak against lasers and applies a significant slowdown.

image

### CODE REVIEW

**Comment by crunchypretzels** [comment]
!review

File: code/modules/clothing/spacesuits/void/misc.dm [hunk 1]
```diff
@@ -125,6 +125,108 @@
...
+		melee = ARMOR_MELEE_MAJOR,
+		bullet = ARMOR_BALLISTIC_CARBINE,
+		laser = ARMOR_LASER_SMALL,
+		energy = ARMOR_ENERGY_MINOR,
+		bomb = ARMOR_BOMB_RESISTANT,
+		bio = ARMOR_BIO_SHIELDED,
+		rad = ARMOR_RAD_SMALL
+	)
+	light_overlay = "helmet_light_himeo_voidsuit"
+	allowed = list(/obj/item/device/flashlight,/obj/item/tank,/obj/item/device/suit_cooling_unit,/obj/item/gun,/obj/item/ammo_magazine,/obj/item/ammo_casing,/obj/item/melee/baton,/obj/item/melee/energy/sword,/obj/item/handcuffs)
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/void/misc.dm:148]:
```suggestion
	allowed = list(
		/obj/item/device/flashlight,
		/obj/item/tank,
		/obj/item/device/suit_cooling_unit,
		/obj/item/gun,
		/obj/item/ammo_magazine,
		/obj/item/ammo_casing,
		/obj/item/melee/baton,
		/obj/item/melee/energy/sword,
		/obj/item/handcuffs
	)
```

File: code/modules/clothing/spacesuits/void/misc.dm [hunk 2]
```diff
@@ -125,6 +125,108 @@
...
+	armor = list(
+		melee = ARMOR_MELEE_MAJOR,
+		bullet = ARMOR_BALLISTIC_CARBINE,
+		laser = ARMOR_LASER_SMALL,
+		energy = ARMOR_ENERGY_MINOR,
+		bomb = ARMOR_BOMB_RESISTANT,
+		bio = ARMOR_BIO_SHIELDED,
+		rad = ARMOR_RAD_SMALL
+	)
+	allowed = list(/obj/item/device/flashlight,/obj/item/tank,/obj/item/device/suit_cooling_unit,/obj/item/gun,/obj/item/ammo_magazine,/obj/item/ammo_casing,/obj/item/melee/baton,/obj/item/melee/energy/sword,/obj/item/handcuffs)
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/void/misc.dm:173]:
```suggestion
	allowed = list(
		/obj/item/device/flashlight,
		/obj/item/tank,
		/obj/item/device/suit_cooling_unit,
		/obj/item/gun,
		/obj/item/ammo_magazine,
		/obj/item/ammo_casing,
		/obj/item/melee/baton,
		/obj/item/melee/energy/sword,
		/obj/item/handcuffs
	)
```

File: code/modules/clothing/spacesuits/void/misc.dm [hunk 3]
```diff
@@ -125,6 +125,108 @@
 	species_restricted = list(BODYTYPE_HUMAN, BODYTYPE_IPC_ZENGHU, BODYTYPE_IPC_BISHOP)
 	refittable = FALSE
 
+/obj/item/clothing/head/helmet/space/void/himeo
+	name = "\improper Himean buzzard voidsuit helmet"
+	desc = "A hardy voidsuit helmet designed specifically for the Himean Planetary Guard. Based on the iconic Vulture voidsuit, and modelled after the Eagle, it boasts improved armor at the cost of weight."
+	desc_extended = "While the Vulture remains the most popular voidsuit in the Coalition, the Free Consortium of Defense and Aerospace Manufacturers has long sought improvements. As the Xanan 'Eagle' remains costly to import, the \
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/void/misc.dm:131]:
```suggestion
	desc_extended = "While the Vulture remains the most popular voidsuit in the Coalition, the Free Consortium of Defense and Aerospace Manufacturers has long sought improvements. As the Xanan \"Eagle\" remains costly to import, the \
```

File: code/modules/clothing/spacesuits/void/misc.dm [hunk 4]
```diff
@@ -125,6 +125,108 @@
...
+	light_overlay = "helmet_light_himeo_voidsuit"
+	allowed = list(/obj/item/device/flashlight,/obj/item/tank,/obj/item/device/suit_cooling_unit,/obj/item/gun,/obj/item/ammo_magazine,/obj/item/ammo_casing,/obj/item/melee/baton,/obj/item/melee/energy/sword,/obj/item/handcuffs)
+	siemens_coefficient = 0.35
+	species_restricted = list(BODYTYPE_HUMAN, BODYTYPE_IPC_INDUSTRIAL, BODYTYPE_IPC_ZENGHU, BODYTYPE_IPC_BISHOP, BODYTYPE_IPC)
+	refittable = FALSE
+
+/obj/item/clothing/suit/space/void/himeo
+	name = "\improper Himean buzzard voidsuit"
+	desc = "A hardy voidsuit designed specifically for the Himean Planetary Guard. Based on the iconic Vulture voidsuit, and modelled after the Eagle, it boasts improved armor at the cost of weight."
+	desc_extended = "While the Vulture remains the most popular voidsuit in the Coalition, the Free Consortium of Defense and Aerospace Manufacturers has long sought improvements. As the Xanan 'Eagle' remains costly to import, the \
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/void/misc.dm:156]:
```suggestion
	desc_extended = "While the Vulture remains the most popular voidsuit in the Coalition, the Free Consortium of Defense and Aerospace Manufacturers has long sought improvements. As the Xanan \"Eagle\" remains costly to import, the \
```

Comment (crunchypretzels) [reply code/modules/clothing/spacesuits/void/misc.dm]: thank you for  all your help gem

**Review by SleepyGemmy — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**